sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -395,7 +395,7 @@ object ScalaNativePluginInternal {
       val paths     = apppaths.map(_.abs) ++ opaths
       val compile   = clangpp.abs +: (flags ++ paths)
 
-      logger.time("Linking native code") {
+      logger.time(s"Linking native code - $gc gc") {
```

Comment:
I'd change the format to `Linking native code ($gc gc)"`.

Reply:
Lets also add similar change to explicitly print current mode in nativeOptimizeNIR:

```
logger.time("Optimizing ($mode mode)") { ... }
```